Prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+"""CP1404 | prac_06 guitars| Darcy Kemp
+program to create guitar objects from user input and append them to a list
+
+I have no idea where to put the estimation, I will put it here.
+
+est: 40
+real: 45
+"""
+from Prac_06.guitar import Guitar
+
+guitars = []
+
+guitar_name = "-1"
+
+while guitar_name != "":
+    guitar_name = input("name: ")
+    if guitar_name != "":
+        guitar_year = int(input("year: "))
+        guitar_cost = int(input("cost: "))
+
+        guitars.append(Guitar(guitar_name, guitar_year, guitar_cost))
+i = 0
+for guitar in guitars:
```

Comment:
you you use enumerate and you wouldn't need the i=0 before, just:[ for i, guitar in enumerate(guitars); ] this will do the exact same thing but then you won't need the i=0 and i+=1 at the end of the loop